### PR TITLE
Syndicate Bomb Buff (THE DRINK NOT THE ACTUAL BOMB)

### PR DESCRIPTION
**Pull request by Therandomhoboo**
# Document the changes in your pull request

Essentially, all other antag based drinks do something for their respective antag roles. This one includes a drink for those special syndicates. This is my first PR so feedback is appreciated

# Wiki Documentation

Two glasses connected with a tank valve. You need to drink from the valve, else it spills. Also if you're part of the mighty Syndicate, it heals for 0.5 Burn & Brute in all body parts per tick

# Changelog

:cl:  

tweak: Syndicate Bomb, heals Syndicate Antags

/:cl:

### CODE REVIEW

**Comment by Maia-J** [comment]
Everyone stay away from this this is my code review I taught him how to make PRs I will review it

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -1106,6 +1106,9 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_icon_state = "syndicatebomb"
 	glass_name = "Syndicate Bomb"
 	glass_desc = "A syndicate bomb."
+	if(is_syndicate(M))
+		M.heal_overall_damage(0.5, 0.5)
+	return ..()
```
Comment (Maia-J) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: This should be moved to the on_mob_life

**Comment by cuackles** [comment]
You should maybe rename it to mention that is the drink cus i was really wondering for a second why would a explosive heal someone

**Comment by cuackles** [comment]
Also might wanna make it so it only heals nukies and maybe infiltrators cus someone would probably try to force someone to prove that they are traitor or something

**Comment by Maia-J** [comment]
> Also might wanna make it so it only heals nukies and maybe infiltrators cus someone would probably try to force someone to prove that they are traitor or something

We can already use other drings to metacheck stuff like revs, clockies, culties, etc. Nobody does it anyway and if anything, admin issue

**Comment by Therandomhoboo** [comment]
> You should maybe rename it to mention that is the drink cus i was really wondering for a second why would a explosive heal someone

I've updated the name to do this

**Comment by cuackles** [comment]
> > Also might wanna make it so it only heals nukies and maybe infiltrators cus someone would probably try to force someone to prove that they are traitor or something
> 
> We can already use other drings to metacheck stuff like revs, clockies, culties, etc. Nobody does it anyway and if anything, admin issue

Oh yeah fair enough, i was thinking with traitors it would be more of an issue but if so many already do it then it should be very few happenings

**Comment by MajManatee** [comment]
If anyone read the title and though "Why would an explosive heal you" then they are dumb and chillean 😎 
fr tho sounds good. All antags should have one.

**Comment by Cartlord** [comment]
If this is was closed because of MapDiffBot, you're fine - that thing's fucked on multiple repos right now.

**Pull request closed without merging**